containers/test/test_container_rpc.sh

```diff
@@ -22,6 +22,7 @@ then
 fi
 
 success=false
+rpc_service_port=17000  # for test only
```

Comment:
How come you use this port instead of the default 7000?

Reply:
I found a collision between `run_rpc_container_tests` and `run_frontend_tests` when executing unittests in parallel. The RPCService of the TaskExecutor in `run_frontend_tests` connected with the RPCService of the RPCTestContainer in `run_rpc_container_tests`!

To resolve this collision, I changed the rpc service port in `run_rpc_container_tests` from 7000 to 17000.